impl/src/parse.rs

```diff
@@ -96,17 +99,33 @@ impl Parse for TraitArgs {
         input.parse::<Token![=]>()?;
         let tag: LitStr = input.parse()?;
         if input.is_empty() {
-            return Ok(TraitArgs::Internal { tag });
+            return Ok(TraitArgs::Internal { tag, default_variant: None });
         }
         input.parse::<Token![,]>()?;
         if input.is_empty() {
-            return Ok(TraitArgs::Internal { tag });
+            return Ok(TraitArgs::Internal { tag, default_variant: None });
+        }
+        if input.peek::<>(kw::default_variant) {
+            input.parse::<kw::default_variant>()?;
+            input.parse::<Token![=]>()?;
+            let default_variant: LitStr = input.parse()?;
+            input.parse::<Option<Token![,]>>()?;
+            if input.is_empty() {
+                return Ok(TraitArgs::Internal { tag, default_variant: Some(default_variant) });
+            }
         }
         input.parse::<kw::content>()?;
```

Comment:
Some of this attribute parsing logic looks incorrect. For example it fails to parse valid attributes such as:

```rust
    #[typetag::serde(
        tag = "type",
        content = "content",
    )]
```

(which is a regression) and fails to reject invalid attributes such as:

```rust
    #[typetag::serde(tag = "type", default_variant = "default" content = "content")]
```